src/applications/appeals/10182/components/ContestableIssuesWidget.jsx

```diff
@@ -96,6 +100,24 @@ const ContestableIssuesWidget = props => {
     [editState],
   );
 
+  useEffect(() => {
+    // contestedIssues becomes undefined after a new save-in-progress loads
+    // (prefill) and removes formData.contestedIssues added by FormApp
+    // Eventually, we'll move all the API-loading & updating code on to the
+    // contestable issues page and remove it all from FormApp
+    if (formData.contestedIssues === undefined) {
+      setFormData({
+        ...formData,
+        contestedIssues: getEligibleContestableIssues(
+          contestableIssues?.issues,
+          {
+            showPart3,
+          },
+        ),
+      });
+    }
+  });
```

Comment:
do we need to add the dependency array param here?

Reply:
I didn't add one, since it only needs to be called initially.

Reply:
Oops, I should have added an empty array :embarrassed: